frontend/conf/dev.conf

```diff
@@ -18,7 +18,7 @@ include "PROD"
 
 identity.webapp.url="https://profile.thegulocal.com"
 identity.production.keys=false
-identity.api.url="https://idapi.thegulocal.com"
+identity.api.url="https://idapi.code.dev-theguardian.com"
```

Comment:
Although we originally developed Membership with everything local, including Identity API, we're now recommending that the Identity version can be the one hosted on CODE: https://docs.google.com/a/guardian.co.uk/spreadsheets/d/1EBU0VrRZCslq_A_mKlocxOhAtCGXI3Ye-KJlKVCIVi0/edit?usp=sharing